src/engine/BMInterfaceGame.php

```diff
@@ -49,23 +48,43 @@ public function create_game(
         }
 
         try {
-            $gameId = $this->insert_new_game($playerIdArray, $maxWins, $description, $previousGameId);
+            if (!isset($currentPlayerId)) {
+                if ($this->isTest) {
```

Comment:
Hmm.  Since `$currentPlayerId` is an argument to `create_game()`, why can't test code set that argument to be the same as `$playerIdArray[0]` and pass it, rather than asking non-test code to do that?